src/main/kotlin/no/nav/familie/oppdrag/repository/OppdragLager.kt

```diff
@@ -5,45 +5,46 @@ import no.nav.familie.kontrakter.felles.oppdrag.Utbetalingsoppdrag
 import no.nav.familie.kontrakter.felles.oppdrag.behandlingsIdForFørsteUtbetalingsperiode
 import no.nav.familie.oppdrag.domene.OppdragId
 import no.nav.familie.oppdrag.iverksetting.OppdragMapper
-import no.nav.familie.oppdrag.iverksetting.Status
 import no.trygdeetaten.skjema.oppdrag.Oppdrag
 import org.springframework.data.relational.core.mapping.Column
 import java.time.LocalDateTime
 
-data class OppdragProtokoll(val fagsystem: String,
-                            @Column("person_ident") val personIdent: String,
-                            @Column("fagsak_id") val fagsakId: String,
-                            @Column("behandling_id") val behandlingId: String,
-                            @Column("input_data") val inputData: String,
-                            val melding: String,
-                            val status: OppdragProtokollStatus = OppdragProtokollStatus.LAGT_PÅ_KØ,
-                            @Column("avstemming_tidspunkt") val avstemmingTidspunkt: LocalDateTime,
-                            @Column("opprettet_tidspunkt") val opprettetTidspunkt: LocalDateTime = LocalDateTime.now()) {
+data class OppdragLager(val fagsystem: String,
+                        @Column("person_ident") val personIdent: String,
+                        @Column("fagsak_id") val fagsakId: String,
+                        @Column("behandling_id") val behandlingId: String,
+                        @Column("input_data") val inputData: String,
+                        val melding: String,
+                        val status: OppdragStatus = OppdragStatus.LAGT_PÅ_KØ,
+                        @Column("avstemming_tidspunkt") val avstemmingTidspunkt: LocalDateTime,
+                        @Column("opprettet_tidspunkt") val opprettetTidspunkt: LocalDateTime = LocalDateTime.now(),
+                        @Column val kvitteringsmelding: String?) {
 
     companion object {
-        fun lagFraOppdrag(utbetalingsoppdrag: Utbetalingsoppdrag, oppdrag: Oppdrag): OppdragProtokoll {
-            return OppdragProtokoll(
+        fun lagFraOppdrag(utbetalingsoppdrag: Utbetalingsoppdrag, oppdrag: Oppdrag): OppdragLager {
+            return OppdragLager(
                     personIdent = utbetalingsoppdrag.aktoer,
                     fagsystem = utbetalingsoppdrag.fagSystem,
                     fagsakId = utbetalingsoppdrag.saksnummer,
                     behandlingId = utbetalingsoppdrag.behandlingsIdForFørsteUtbetalingsperiode(),
                     avstemmingTidspunkt = utbetalingsoppdrag.avstemmingTidspunkt,
                     inputData = ObjectMapper().writeValueAsString(utbetalingsoppdrag),
-                    melding = ObjectMapper().writeValueAsString(oppdrag)
+                    melding = ObjectMapper().writeValueAsString(oppdrag),
```

Comment:
'melding' bør da kalles 'utgaaende_oppdrag' og tilsvarende for 'inputData'...?

Reply:
Good catch, ser jeg har glemt å oppdatere disse 😅